vimiv/api/working_directory.py

```diff
@@ -193,14 +194,19 @@ def _maybe_emit_image_changed(self) -> None:
         _logger.debug("Image file updated")
         status.update("image file changed")
 
-    def _emit_changes(self, images: List[str], directories: List[str]) -> None:
+    def _emit_changes(
+        self,
+        images_meta: List[Tuple[str, os.stat_result]],
+        directories_meta: List[Tuple[str, os.stat_result]]
+    ) -> None:
         """Emit changed signals if the content in the directory has changed.
 
         Args:
-            images: Updated list of images in the working directory.
-            directories: Updated list of directories in the working directory.
+            images_meta: Updated list of images in the working directory.
+            directories_meta: Updated list of directories in the working directory.
         """
         # Image filelist has changed, relevant for thumbnail and image mode
+        images, directories = files.strip_meta((images_meta, directories_meta))
```

Comment:
This links to the comment made in the library, we end up with calls that expect the meta but don't get it.